.travis.yml

```diff
@@ -4,9 +4,10 @@ php:
   - '7.1'
   - '7.2'
   - '7.3'
+  - '7.4'
 install:
-  - composer update
+  - composer install
```

Comment:
It was a mistake that the `composer.lock` file was committed. Could you remove `composer.lock` and change this back to `composer update`.

Reply:
Actually, it's fine to keep it as `composer install`, but `composer.lock` should not be there.

Reply:
I think it should remove `composer.lock` and keep `composer install` command existing :).